web/forms.py

```
@@ -5,13 +5,16 @@
 from django.db import IntegrityError
 from django.utils.crypto import get_random_string
 from django.utils.text import slugify
+from django.core.exceptions import ValidationError
```

Ruff (0.8.2)

`django.core.exceptions.ValidationError` imported but unused

Remove unused import: `django.core.exceptions.ValidationError`

(F401)